### ISSUE TITLE
Validation Loss

### ISSUE THREAD

**Issue by Ysmita** [fregu856/segmentation#12]
Hi @fregu856 
Inspite of training the model for 200 epochs the training loss is fluctuating between 4.9-5, and validation loss is fluctuating between 7.5-8.2 .   The model is not working on random images, it just works on cityscapes dataset. How to reduce the loss, how to calculate the accuracy and mIoU for the training and validation set and also how to make the model more generic?

Thanking you in anticipation
